src/network/connection.h

```diff
@@ -520,7 +524,10 @@ class Peer {
 		void ResetTimeout()
 			{MutexAutoLock lock(m_exclusive_access_mutex); m_timeout_counter = 0.0; };
 
-		bool isTimedOut(float timeout);
+		bool isHalfOpen() const { return m_half_open; }
+		void SetFullyOpen() { m_half_open = false; }
```

Comment:
The naming scheme is all over place in this code, do really want me to fix this one instance?

Reply:
The class uses both - camel case and pascal case. It would be more consistent with the rest of the code in the long turn if code style rules are applied to new or changed functions.
I'll leave the decision up to you. I won't block a PR just for such reason.